hudi-client/hudi-client-common/src/main/java/org/apache/hudi/io/HoodieWriteHandle.java

```diff
@@ -112,8 +115,21 @@ private String makeWriteToken() {
     return FSUtils.makeWriteToken(getPartitionId(), getStageId(), getAttemptId());
   }
 
+  protected StoragePath getPartitionPath(String partitionPath) {
```

Comment:
nit: we can change the method name to `toPhysicalPath`

Reply:
make sense!